base/test/h264encoderv4l2_tests.cpp

```diff
@@ -71,13 +76,13 @@ BOOST_AUTO_TEST_CASE(rgb24_1280x720, *boost::unit_test::disabled())
 	auto encoder = boost::shared_ptr<Module>(new H264EncoderV4L2(encoderProps));
 	copy->setNext(encoder);
 
-	auto fileWriter = boost::shared_ptr<Module>(new FileWriterModule(FileWriterModuleProps("./data/testOutput/Raw_RGB24_1280x720.h264", true)));
-	encoder->setNext(fileWriter);
+	auto sink = boost::shared_ptr<ExternalSinkModule>(new ExternalSinkModule());
+	encoder->setNext(sink);
```

Comment:
why cudamemcpy ? - remove this test if not useful

Reply:
This module accepts both host and cuda memory